lib/delayed_job_worker_pool/registry.rb

```diff
@@ -13,12 +15,12 @@ def include_worker?(pid)
       worker_pids.include?(pid)
     end
 
-    def has_workers?
+    def workers?
```

Comment:
Was this method a public interface? Would the rename count as a breaking change?

Reply:
It's an internal interface.